.github/workflows/ci.yml

```diff
@@ -358,7 +358,7 @@ jobs:
           ./src/univalue/unitester.exe
 
       - name: Run benchmarks
-        run: ./bin/bench_bitcoin.exe -sanity-check -priority-level=high
+        run: ./bin/bench_bitcoin.exe -sanity-check
```

Comment:
Why not drop `-priority-level=high` here as well:https://github.com/bitcoin/bitcoin/blob/7a3afe6787bc36fd98684eac020f9abdbfae6f0a/src/bench/CMakeLists.txt#L82-L84 ?

Will it have a significant impact on execution time?

Reply:
I'm just changing CI, so that we catch regressions. Someone could follow up with changing this for all developers & users.

Reply:
But this introduces another inconsistency between Windows and other platforms. Initially, `run: ./bin/bench_bitcoin.exe -sanity-check -priority-level=high` was written to mimic the `bench_sanity_check_high_priority` test task behaviour.

Reply:
Can you explain your issue with running more tests in this CI?

Reply:
> The point of this change is to immediately start catching regressions in at least one CI.

Ok. Resolved.